test/token/ERC20/RecoverableToken.test.js

```diff
@@ -0,0 +1,39 @@
+const StandardTokenMock = artifacts.require('StandardTokenMock');
+const RecoverableTokenMock = artifacts.require('RecoverableTokenMock');
+const BigNumber = web3.BigNumber;
+
+require('chai')
+  .use(require('chai-as-promised'))
+  .use(require('chai-bignumber')(BigNumber))
+  .should();
+
+contract('RecoverableToken', function ([creator, owner, recipient, anotherAccount]) {
+  beforeEach(async function () {
+    this.token = await StandardTokenMock.new(owner, 100);
+    this.otherToken = await RecoverableTokenMock.new(anotherAccount, 100);
+  });
+
+  describe('Recover token', function () {
+    it('transfer token from contract to owner', async function () {
```

Comment:
Could you add a couple more test cases? In particular, I'd like to see the function call fail if the caller is not `owner`.

Reply:
ok but if I do this i'm testing owner contract, you don't think ?

Reply:
Not really, that call failing is part of the specification of your contract. In any case, you'd be testing that you've correctly used the `onlyOwner` modifier :)